bin/portageq

```diff
@@ -1451,9 +1451,11 @@ try:
 
     def main(argv):
         argv = portage._decode_argv(argv)
-
-        nocolor = os.environ.get("NOCOLOR")
-        if nocolor in ("yes", "true"):
+        if os.environ.get("NO_COLOR"):
```

Comment:
If we put all of this in one function, we can add a nice comment about how we can cleanup NOCOLOR support (not NO_COLOR) in a few years.